app/core/voyagerUtils.js

```diff
@@ -38,56 +38,77 @@ module.exports = {
   },
 
   /**
-   * Gets the next level original id for a given level
+   * Gets the next level original ids for a given level
    * @param {Object} level - The level object.
    * @param {Object[]} level.nextLevels - The array of nextLevels for the given level.
-   * @returns {string|undefined} - 'Original' id of the next level, or undefined.
+   * @returns {string[]]} - Array of next level 'Original' ids.
    */
   getNextLevelOriginalForLevel: function (level) {
-    let nextLevels = level.nextLevels
+    const nextLevels = level.nextLevels
+    const nextLevelOriginals = []
     if ((nextLevels || []).length > 0) {
-      return nextLevels[0].levelOriginal // assuming that there will be just one next level for voyager v1 as of now.
+      nextLevelOriginals.push(nextLevels[0].levelOriginal) // assuming that there will be just one next level for voyager v1 as of now.
```

Comment:
Fine to leave as is. Mostly asking for my understanding. As we're getting the next level original of only the first level it feels like the code could avoid the conditional and nextLevelOriginals resulting array. It could also be a map over the nextLevels, returning the originals.

Something like:

```
(level.nextLevels || [])
  .map(level => level.levelOriginal)
```

Although from your comment the design appears intentional. Is there any additional details regarding how this method will change when the 1FH capstone level is added?

Reply:
Thats a great suggestion. With the addition of 1FH capstone flow, we would most likely loop over `levels.nextLevels` to find the next level original (since the capstone will have multiple next levels, and only one will be returned based on the state of capstone level).
Will update it based on your suggestion when I will add the logic for capstone flow.